Master.py

```diff
@@ -1089,27 +1055,42 @@ def regression_coefficient(paths, param, tech):
 
     # Reads the files present in input folder and extract hub_heights represented.
 
-    inputfiles = glob.glob(paths[tech]["TS_height"] + '_*')
+    inputfiles = glob.glob(paths[tech]["TS_param"] + '_*')
     if len(inputfiles) == 0:
         reg_miss_files()
         timecheck('End')
         return
-    elif len(inputfiles) == 1:
-        hub_heights = np.zeros(1, dtype=int)
     else:
-        hub_heights = np.zeros(len(inputfiles), dtype=int)
+        settings = np.zeros(len(inputfiles), dtype=int)
 
         h = 0
         for filename in inputfiles:
-            heigh = int(filename.replace(paths[tech]["TS_height"] + '_', '').replace('_TS_' + str(param["year"])+ '.csv', ''))
```

Comment:
don't you mean height instead of heigh?

Reply:
Yes, it was a typo. Now that we account for the orientation of the PV too, I call it settings or set. The new name might be a bit confusing, any ideas ?